<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AppCafe Configuration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="SoftwareManager.qrc">
    <normaloff>:/icons/appcafe.png</normaloff>:/icons/appcafe.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_config">
      <attribute name="icon">
       <iconset resource="SoftwareManager.qrc">
        <normaloff>:/icons/preferences-system.png</normaloff>:/icons/preferences-system.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Configuration</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="group_install">
         <property name="title">
          <string>Post-Install Actions</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QCheckBox" name="check_desktop">
               <property name="text">
                <string>Create Desktop Icons</string>
               </property>
               <property name="icon">
                <iconset resource="SoftwareManager.qrc">
                 <normaloff>:/icons/xdg_desktop.png</normaloff>:/icons/xdg_desktop.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="check_menu">
               <property name="text">
                <string>Create Menu Icons</string>
               </property>
               <property name="icon">
                <iconset resource="SoftwareManager.qrc">
                 <normaloff>:/icons/xdg_menu.png</normaloff>:/icons/xdg_menu.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="check_mime">
               <property name="text">
                <string>Register File Associations</string>
               </property>
               <property name="icon">
                <iconset resource="SoftwareManager.qrc">
                 <normaloff>:/icons/xdg_mime.png</normaloff>:/icons/xdg_mime.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="check_paths">
               <property name="text">
                <string>Create Path Links</string>
               </property>
               <property name="icon">
                <iconset resource="SoftwareManager.qrc">
                 <normaloff>:/icons/xdg_paths.png</normaloff>:/icons/xdg_paths.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="group_download">
         <property name="title">
          <string>Keep Downloaded Files</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QFrame" name="frame_dldir">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Download Dir</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="line_downloadDir"/>
             </item>
             <item>
              <widget class="QToolButton" name="tool_getDownloadDir">
               <property name="text">
                <string>Download Directory</string>
               </property>
               <property name="icon">
                <iconset resource="SoftwareManager.qrc">
                 <normaloff>:/icons/folder.png</normaloff>:/icons/folder.png</iconset>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonIconOnly</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_repos">
      <attribute name="icon">
       <iconset resource="SoftwareManager.qrc">
        <normaloff>:/icons/app_needroot.png</normaloff>:/icons/app_needroot.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Repositories</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QComboBox" name="combo_repo"/>
         </item>
         <item>
          <widget class="QToolButton" name="tool_repo_add">
           <property name="text">
            <string>Add</string>
           </property>
           <property name="icon">
            <iconset resource="SoftwareManager.qrc">
             <normaloff>:/icons/add-root.png</normaloff>:/icons/add-root.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonIconOnly</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tool_repo_remove">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="SoftwareManager.qrc">
             <normaloff>:/icons/remove-root.png</normaloff>:/icons/remove-root.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="group_repomirrors">
         <property name="title">
          <string>Repository Mirrors</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="2">
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QToolButton" name="tool_repomirror_up">
              <property name="text">
               <string>Move Up</string>
              </property>
              <property name="icon">
               <iconset resource="SoftwareManager.qrc">
                <normaloff>:/icons/arrow-up.png</normaloff>:/icons/arrow-up.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="tool_repomirror_down">
              <property name="text">
               <string>Move Down</string>
              </property>
              <property name="icon">
               <iconset resource="SoftwareManager.qrc">
                <normaloff>:/icons/arrow-down.png</normaloff>:/icons/arrow-down.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="tool_repomirror_add">
              <property name="text">
               <string>Add</string>
              </property>
              <property name="icon">
               <iconset resource="SoftwareManager.qrc">
                <normaloff>:/icons/add-root.png</normaloff>:/icons/add-root.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonIconOnly</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="tool_repomirror_remove">
              <property name="text">
               <string>Remove</string>
              </property>
              <property name="icon">
               <iconset resource="SoftwareManager.qrc">
                <normaloff>:/icons/remove-root.png</normaloff>:/icons/remove-root.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonIconOnly</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <widget class="QListWidget" name="list_repomirrors"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="SoftwareManager.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
